reset_password.cgi

```diff
@@ -49,7 +48,14 @@ if ($cgi->param('do_save')) {
     if ($old_password eq $password_1) {
         ThrowUserError('new_password_same');
     }
-    validate_password($password_1, $password_2);
+
+    my $pwqc = Bugzilla->passwdqc;
+    if (not $pwqc->validate_password($password_1)) {
+        ThrowUserError("password_insecure", { reason => $pwqc->reason });
+    }
+    elsif ($password_1 ne $password_2) {
+        ThrowUserError("password_mismatch");
+    }
```

Comment:
No, it is intentional to check the password sameness second, as passwdqc also lets us set a max password. It is useful to avoid comparing huge strings.